ReactWindows/ReactNative.Shared.Tests/Modules/Network/NetworkingModuleTests.cs

```diff
@@ -10,36 +10,42 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+#if WINDOWS_UWP
 using Windows.Web.Http;
+#else
+using System.Net;
+using System.Net.Http;
+using HttpStreamContent = System.Net.Http.StreamContent;
+#endif
```

Comment:
I don't think this is needed, we were only using it when HttpStreamContent was an output of a method.  Here, you're just making constructor calls isolated in #ifs anyway, so its better not to type alias.